code/FulpstationCode/fulp_ops.dm

```diff
@@ -0,0 +1,107 @@
+/datum/outfit/syndicate/plasmaman
+	name = "Syndicate Operative Plasmaman - Basic"
+	head = /obj/item/clothing/head/helmet/space/plasmaman/robotics/syndicate
+	uniform = /obj/item/clothing/under/plasmaman/security/syndicate
+	r_hand= /obj/item/tank/internals/plasmaman/belt/full
+	mask = /obj/item/clothing/mask/gas/syndicate
+	gloves = /obj/item/clothing/gloves/combat
+	backpack_contents = list(/obj/item/paper/guides/plasmemeops)
+
+/datum/outfit/syndicate/leader/plasmaman
+	name = "Syndicate Leader Plasmaman - Basic"
+	head = /obj/item/clothing/head/helmet/space/plasmaman/robotics/syndicate
+	uniform = /obj/item/clothing/under/plasmaman/security/syndicate
+	r_pocket= /obj/item/tank/internals/plasmaman/belt/full // They spawn with both hands busy, so they have to manually set internals
+	mask = /obj/item/clothing/mask/gas/syndicate
+	id = /obj/item/card/id/syndicate/nuke_leader
+	gloves = /obj/item/clothing/gloves/krav_maga/combatglovesplus
+	command_radio = TRUE
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival, /obj/item/paper/guides/plasmemeops)
+
+/obj/item/clothing/head/helmet/space/plasmaman/robotics/syndicate
+	name = "syndicate plasma envirosuit helmet"
+	desc = "A plasmaman envirosuit helmet designed for syndicate operatives."
+
+/obj/item/clothing/under/plasmaman/security/syndicate
+	name = "syndicate plasma envirosuit"
+	desc = "A non-descript and slightly suspicious looking plasmaman enviromental suit."
+	worn_icon = 'icons/fulpicons/phoenix_nest/syndicate_suits.dmi'
+	icon = 'icons/fulpicons/phoenix_nest/syndicate_suits_icons.dmi'
+	icon_state = "syndicate_envirosuit"
+	has_sensor = NO_SENSORS
+	armor = list("melee" = 10, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 100, "rad" = 0, "fire" = 95, "acid" = 95)
+
+// Plasmeme syndi hardsuit
+
+/obj/item/clothing/head/helmet/space/hardsuit/syndiplasmeme
+	name = "plasmaman blood-red hardsuit helmet"
+	desc = "An advanced helmet designed for work in special operations. Property of the Plasmaman Division of Gorlex Marauders."
```

Comment:
is it "the Gorlex Marauders" or just "Gorlex Marauders"? I dunno, and I'm too lazy to check for myself, but you might wanna check existing mentionings of them in the code, just to be sure.

Reply:
its simply "Gorlex Marauders" taking from the same description the blood-red hardsuit has